routes/web.php

```diff
@@ -17,4 +17,5 @@
 
 Auth::routes();
 
-Route::get('/home', 'HomeController@index')->name('home');
+// Route::get('/home', 'HomeController@index')->name('home');
+Route::resource('home', 'HomeController');
```

Comment:
this works but it has also added routes for a bunch of other non-existent things like `create`, `edit`, `update`, `destroy`.

It also assumes your model is named `home` instead of `post`.

I'd either rename the resource to `posts` and use `->only()`, or i'd just add `Route::post('home', 'HomeController@store')`.